src/main/java/com/minecolonies/coremod/colony/crafting/CustomRecipeFactory.java

```diff
@@ -33,15 +31,24 @@ public class CustomRecipeFactory implements IFactory<FactoryVoidInput, CustomRec
 {
     private final static String CUSTOM_RECIPE_ID_PROP = "id";
 
+    private static boolean checkContext(final Object[] context, final Class<?>... types)
```

Comment:
Is this not a duplicate from the other one you added?

Reply:
Good catch.  In fact neither of these ended up being used -- I had some early code to support distinguishing between multiple sets of different parameter types, but in the end it was simpler to just change all the callers to use the new args.